uwg_2023/uwg_2023_stac.ipynb

```diff
@@ -0,0 +1,850 @@
+       "    <div style=\"margin-left: 48px;\">\n",
+       "        <h4 style=\"margin-bottom: 0px;\">Link:  stac-fastapi </h4>\n",
+       "        <table style=\"width: 100%; text-align: left;\">\n",
+       "            <tr><td style=\"text-align: left;\"><strong>rel:</strong> root </td></tr>\n",
+       "            <tr><td style=\"text-align: left;\"><strong>href:</strong> https://stac.asf.alaska.edu/ </td></tr>\n",
+       "            \n",
+       "                <tr><td style=\"text-align: left;\"><strong>type:</strong> application/json </td></tr>\n",
+       "            \n",
+       "            \n",
+       "                <tr><td style=\"text-align: left;\"><strong>title:</strong> stac-fastapi </td></tr>\n",
+       "            \n",
+       "            \n",
+       "        </table>\n",
+       "        \n",
+       "    </div>\n",
+       "</div>\n",
+       "            \n",
+       "                \n",
+       "\n",
+       "<div class=\"jp-RenderedHTMLCommon jp-RenderedHTML jp-mod-trusted jp-OutputArea-output\">\n",
+       "    <div style=\"\n",
+       "        width: 24px;\n",
+       "        height: 24px;\n",
+       "        background-color: #FFF7E5;\n",
+       "        border: 3px solid #FF6132;\n",
+       "        border-radius: 5px;\n",
+       "        position: absolute;\">\n",
+       "    </div>\n",
+       "    <div style=\"margin-left: 48px;\">\n",
+       "        <h4 style=\"margin-bottom: 0px;\">Link: </h4>\n",
+       "        <table style=\"width: 100%; text-align: left;\">\n",
+       "            <tr><td style=\"text-align: left;\"><strong>rel:</strong> self </td></tr>\n",
+       "            <tr><td style=\"text-align: left;\"><strong>href:</strong> https://stac.asf.alaska.edu/collections/sentinel-1-global-coherence/items/N45W086_summer_vv_COH24 </td></tr>\n",
+       "            \n",
+       "                <tr><td style=\"text-align: left;\"><strong>type:</strong> application/geo+json </td></tr>\n",
+       "            \n",
+       "            \n",
+       "            \n",
+       "        </table>\n",
+       "        \n",
+       "    </div>\n",
+       "</div>\n",
+       "            \n",
+       "        </details>\n",
+       "    \n",
+       "        </details>\n",
+       "    </div>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "<Item id=N45W086_summer_vv_COH24>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bbox = [-85.8173, 43.7597, -85.1838, 44.2131]  # [min_lon, min_lat, max_lon, max_lat]\n",
+    "start_date = datetime(2020, 5, 1)\n",
+    "end_date = datetime(2020, 6, 1)\n",
+    "search = client.search(\n",
+    "    max_items=10,\n",
+    "    collections=['sentinel-1-global-coherence'],\n",
+    "    bbox=bbox,\n",
+    "    datetime=(start_date, end_date)\n",
+    ")\n",
+    "coh24 = [item for item in search.items() if 'COH24' in item.id][0]\n",
```

Comment:
you should be able to filter for COH24 up front in the search; let me recall which attribute we populated for that and how it interacts with `client.search`

Reply:
`search = client.search(max_items=10, collections=['sentinel-1-global-coherence'], ...., query=['sar:product_type=COH24'])`

The `summaries` field on the collection record is intended to summarize the properties and values you can search for (e.g. `sar:product_type=AMP`, `season=fall`, ...)

Reply:
I'll likely implement this after UWG, since I'd like to ensure that I don't make any breaking changes day of.